backend/tests/unit/dal/Services/PropertyServiceTest.cs

```diff
@@ -13,102 +15,95 @@ namespace Pims.Dal.Test.Services
 {
     [Trait("category", "unit")]
     [Trait("category", "dal")]
-    [Trait("area", "admin")]
-    [Trait("group", "lease")]
+    [Trait("group", "property")]
     [ExcludeFromCodeCoverage]
-    public class LeaseServiceTest
```

Comment:
The LeaseServiceTests were in a file called PropertyServiceTest. That has been corrected above. Also added basic tests for the PropertyService here that were removed when Properties were refactored in September.